src/main/java/seedu/address/model/person/UniqueModuleList.java

```diff
@@ -13,7 +13,7 @@
 public class UniqueModuleList implements Iterable<Mod> {
 
 
-    private static final ObservableList<Mod> internalList = FXCollections.observableArrayList();
+    private ObservableList<Mod> internalList = FXCollections.observableArrayList();
```

Comment:
Just a note, if this were to be used to show in the UI, this should be final so that FXML refers to the same object as is being stored here

Reply:
Do we need the module list for the UI part?

Reply:
Up to you if you think that is necessary or not! I think its okay to leave it out